diff --git a/drivers/video/samsung_extdisp/s3cfb_extdsp.h b/drivers/video/samsung_extdisp/s3cfb_extdsp.h
index 99f134c..3927c1a 100644
--- a/drivers/video/samsung_extdisp/s3cfb_extdsp.h
+++ b/drivers/video/samsung_extdisp/s3cfb_extdsp.h
@@ -151,7 +151,7 @@ struct s3cfb_extdsp_user_window {
 #define S3CFB_EXTDSP_GET_FREE_BUFFER		_IOW('F', 329, unsigned int)
 
 extern struct fb_ops			s3cfb_extdsp_ops;
-extern inline struct s3cfb_extdsp_global	*get_extdsp_global(int id);
+//extern inline struct s3cfb_extdsp_global	*get_extdsp_global(int id);
 
 /* S3CFB_EXTDSP */
 extern int s3cfb_extdsp_enable_window(struct s3cfb_extdsp_global *fbdev, int id);
